SGoopas/Assets/Scripts/Game/2D/2D Player States/JumpingRight2D.cs

```diff
@@ -6,16 +6,29 @@ namespace PlayerStates
 {
     public class JumpingRight2D : Base2DState
     {
-        public JumpingRight2D(BasePlayerState previousState) : base(previousState) {}
-        public JumpingRight2D(GameObject player, MasterPlayerStateMachine playerStateMachine, Transform groundCheck) : base(player, playerStateMachine, groundCheck) {}
-       
+        public JumpingRight2D(BasePlayerState previousState) : base(previousState) {

```

Comment:
uhh this is kinda sketch... Can you include a comment explaining this?

Reply:
She can turn around in midair. She couldn't before. Otherwise movement is unaffected. Lets you dash left and right in the air, and see which direction you'll dash in. The constructor now has the flip sprite method to make sure she's visually facing the same way as the state. The constructor didn't have it before because you always entered jumping from a grounded state where she was facing the right way, but now you can enter it from the other jumping state

Reply:
yeah a comment explaining that would be nice. Or maybe just rename FlipSprite() to MakeSpriteFaceRight to be more clear. I was confused, I thought it was arbitrarily flipping the sprite direction 😮

Reply:
done